README.md

```diff
@@ -89,9 +89,11 @@ use({
     chat_dir = vim.fn.stdpath("data"):gsub("/$", "") .. "/wtf/chats",
     -- Default AI popup type
     popup_type = "popup" | "horizontal" | "vertical",
+    -- Set custom OpenAI compatible API url  (Ollama example: "http://ollama.example.com:11434/v1/")
+    openai_api_url = nil,
     -- An alternative way to set your API key
     openai_api_key = "sk-xxxxxxxxxxxxxx",
-    -- ChatGPT Model
+    -- ChatGPT Model  (Ollama example: "qwen2:7b")
```

Comment:
wanted to make this explicit because model id in ollama have some special characters